docs/audit.md

```diff
@@ -6,6 +6,20 @@ a pre-generated baseline. Some common use cases of this include:
 -   **Manually labelling secrets**, to distinguish between true and false positives
 -   **Comparing baselines**, to determine the effect of a certain configuration
```

Comment:
Thanks for making the change! Could you move the section above under its own heading (named something along the lines of `What It Is`?) below the TOC?

So rather than
<img width="1093" alt="Screen Shot 2021-12-23 at 11 34 39 AM" src="https://user-images.githubusercontent.com/12804086/147268485-ccbacf3c-95af-4aa2-8ebe-f37dfe36c9a5.png">

have it align more with
<img width="1080" alt="Screen Shot 2021-12-23 at 11 34 34 AM" src="https://user-images.githubusercontent.com/12804086/147268522-804cf6b7-3d82-4467-a71e-bfe2c3322f68.png">

so that the TOC is directly below the main heading

Reply:
3e49973